host_v2/src/pages/GameInProgress.tsx

```diff
@@ -39,20 +112,42 @@ export default function GameInProgress() {
         return footerButtonTextDictionary[statePosition];
       };
 
+      const handleConfidenceGraphClick = (selectedIndex: number | null) => {
+        setConfidenceGraphClickIndex(selectedIndex);
+      };
+
+      const handleTimerIsFinished = () => {
+        console.log('timer is finished'); // eslint-disable-line
+      };
+
     return(
-        <FooterContent
-        inputNum={numPlayers} // need # for answer bar
-        totalNum={totalNum} // number of answers
-        footerBar = {currentState}
-        phaseOneTime={phaseOneTime}
-        phaseTwoTime={phaseTwoTime}
-        footerButtonText={getFooterText(
-            teams ? teams.length : 0,
-            totalNum,
-            statePosition,
-        )} // provides index of current state for use in footer dictionary
-        // footerButtonText='End Answering'
+      <StackContainerStyled>
```

Comment:
`GameInProgress` now contains all components that are used during the `gameSession`. This includes the main structure of the page (header, body, footer) and then `GameInProgressContent` is the body that adjusts as the `gameSession.currentState` changes.